src/classes/AuthenticationSession.ts

```diff
@@ -45,41 +108,59 @@ export class AuthenticationSession {
   }
 
   private processPasswordResponse([password]: string[]): void {
+    if (!AuthenticationSession.sftpFusionAuthAppId) {
```

Comment:
As discussed -- since this is required for the class to operate let's just make it part of the constructor and we can elevate these checks to `server.ts` or `index.ts`.